prime-angular-frontend/src/app/modules/paper-enrolment/pages/overview-page/overview-page.component.html

```diff
@@ -15,7 +15,8 @@
 
   <app-demographic-overview [demographic]="enrollee"></app-demographic-overview>
   <app-care-setting-overview [careSettings]="enrollee"></app-care-setting-overview>
-  <app-regulatory-overview [regulatory]="enrollee"></app-regulatory-overview>
+  <app-regulatory-overview [regulatory]="enrollee"
+                           [deviceProviderIdentifier]="enrollee?.deviceProviderIdentifier"></app-regulatory-overview>
```

Comment:
You shouldn't need to pass in two Input bindings into the regulatory review.  The idea of the form  state model is that it matches the keys from enrollee model so you only need the original binding that gets passed the enrollee.

Reply:
fixed

Reply:
Fixed